strawberry/schema/execute.py

```diff
@@ -91,7 +91,7 @@ async def execute(
     return ExecutionResult(
         data=result.data,
         errors=result.errors,
-        extensions=extensions_runner.get_extensions_results(),
```

Comment:
I've changed get_extensions_results_async to get_extensions_results 

and get_extensions_results to get_extensions_results_sync to stick with our conventions :)